src/app.py

```diff
@@ -1,90 +1,886 @@
-# Copyright (C) 2022, Pyronear.
+from datetime import date, datetime
 
-# This program is licensed under the Apache License version 2.
-# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+import dash
+import dash_bootstrap_components as dbc
+import dash_daq as daq
+import pandas as pd
+# Various modules provided by Dash and Dash Leaflet to build the page layout
+from dash import dcc, html
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+from user_agents import parse
 
-import time
-from datetime import date
-from pathlib import Path
+########################################################################################################################
+# Utils ----------------------------------------------------------------------------------------------------------------
 
-import streamlit as st
+# Fetching French departements
+departments = pd.read_csv(
+    'https://static.data.gouv.fr/resources/departements-de-france/20200425-135513/departements-france.csv'
+)
 
-import config
+department_options = []
+departments['to_be_displayed'] = departments['code_departement'] + ' - ' + departments['nom_departement']
+for _, row in departments.iterrows():
+    department_options.append(
+        {
+            'label': row['to_be_displayed'],
+            'value': row['code_departement']
+        }
+    )
+
+# Setting the options for the type of wildfire
+etiquette_options = [
+    {'label': 'Feu de forêt', 'value': 'wildfire'},
+    {'label': 'Feu industriel', 'value': 'industrial_fire'}
+]
+
+
+def is_int(element):
+    try:
+        int(element)
+        return True
+
+    except ValueError:
+        return False
+
+
+def build_image_upload():
+
+    upload = dcc.Upload(
+        id='image-upload',
+        children=html.Div(
```

Comment:
As suggested here https://community.plotly.com/t/uploading-relatively-small-files-is-taking-too-long-in-dash-plotly/32785,
let's temporarily save the image to disk before uploading :+1:

Reply:
Edit done.